cliquet/__init__.py

```diff
@@ -188,6 +191,18 @@ def includeme(config):
     handle_api_redirection(config)
     config.add_tween("cliquet.end_of_life_tween_factory")
 
+    if settings['cliquet.sentry_dsn']:
+        raven_client = raven.Client(
+            settings['cliquet.sentry_dsn'],
+            include_paths=['cornice', 'cliquet'] +
+            settings['cliquet.sentry_projects'].split(','),
+            release=settings['cliquet.project_version'])
+
+        raven_client.captureMessage("%s %s starting." % (
```

Comment:
I think we should ensure we test it's running when the setting is set and not running when it's not. (yeah, me again about tests :p)